scripts/staging/SIMD-double-vectors/systemds

```diff
@@ -167,7 +167,7 @@ Worker Usage: $0 [-r] WORKER [SystemDS.jar] <portnumber> [arguments] [-help]
 
     port         : The port to open for the federated worker.
 
-Federated Monitoring Usage: $0 [-r] FEDMONITOR [SystemDS.jar] <portnumber> [arguments] [-help]
+Federated Monitoring Usage: $0 [-r] FEDMONITORING [SystemDS.jar] <portnumber> [arguments] [-help]
```

Comment:
Done, I removed the line with the federated monitoring.

Reply:
this comment was more because the file is in staging and should probably in general be removed.
therefore we should not start modifying it, but instead remove it later.